ui/src/app/components/app-router.tsx

```diff
@@ -77,43 +65,9 @@ export const withFullHeight =
     );
   };
 
-// This function is invoked if react-router `<Prompt>` is rendered by a component that wants the user to
-// confirm navigating away from the page. The default behavior of <Prompt> is being overridden by this
-// getUserConfirmation function so we can provide a custom styled warning modal instead of the browser's default.
-const getUserConfirmation = (message, callback) => {
-  const modal = document.createElement('div');
-  document.body.appendChild(modal);
-
-  const withCleanup = (answer) => {
-    ReactDOM.unmountComponentAtNode(modal);
-    document.body.removeChild(modal);
-    callback(answer);
-  };
-
-  ReactDOM.render(
-    <Modal>
-      <ModalTitle>Warning!</ModalTitle>
-      <ModalBody>{message}</ModalBody>
-      <ModalFooter>
-        <Button type='link' onClick={() => withCleanup(false)}>
-          Cancel
-        </Button>
-        <Button type='primary' onClick={() => withCleanup(true)}>
-          Discard Changes
-        </Button>
-      </ModalFooter>
-    </Modal>,
-    modal
-  );
-};
-
-export const AppRouter = ({ children }): React.ReactElement => {
-  return (
-    <BrowserRouter getUserConfirmation={getUserConfirmation}>
```

Comment:
don't need this anymore, because it was only used to wrap `getUserConfirmation()`